### PR TITLE
Prometheus: Remove raw query toggle

### LINKED ISSUE #58626: Prometheus: Remove raw query toggle in query builder and always show raw query
What happened:

In the Prometheus query builder, we have a toggle for Raw query that is used to toggle the preview of the query that the query builder produces. During a few user interviews, we've noticed that the query editor has (too) many buttons/toggles, which makes small buttons basically invisible. Moreover, no users we talked to toggle their query preview off.

To reduce the number of buttons/toggles we decided that it would make sense to:

- Remove the Raw query toggle
- Always show raw query (query text)
- Remove the heading/title Raw query.

Screenshot 2022-11-11 at 11 20 05 AM

The Loki Query Builder is doing the same, reference here:
https://github.com/grafana/grafana/issues/58511

### PR DESCRIPTION
**Pull request by gtk-grafana**
**What is this feature?**
This PR removes the raw query toggle from the Prometheus query builder, and reduces the size/contrast of the raw query section.

**Why do we need this feature?**
To simplify the query builder. Seeing the raw query is a good way for users to confirm that values selected in the query builder are as expected, even without knowledge of promql, and the ability to hide/remove this section within the query builder is not useful functionality.

**Who is this feature for?**
First time and existing users of the prometheus query builder.

**Which issue(s) does this PR fix?**:
Fixes https://github.com/grafana/grafana/issues/58626

**Special notes for your reviewer**:
Can anyone think of a use case where we absolutely don't want to show the raw query text? I guess if people really hate seeing the raw query we could move this configuration to elsewhere in the UI?

### CODE REVIEW

**Comment by gtk-grafana** [comment]
@itsmylife this was assigned to you, but I stole it to have a quick win I could work on while jet-lagged. Did I miss anything that you and Diego had discussed?

**Review by itsmylife — APPROVED** [review]
LGTM but just a small question

File: public/app/plugins/datasource/prometheus/querybuilder/components/QueryPreview.tsx [hunk 1]
```diff
@@ -13,9 +13,7 @@ export function QueryPreview({ query }: Props) {
   return (
     <EditorRow>
       <EditorFieldGroup>
-        <EditorField label="Raw query">
```
[the remark this answers is not in the document]
Comment (gtk-grafana) [reply public/app/plugins/datasource/prometheus/querybuilder/components/QueryPreview.tsx:16]: It was the intention, and one of the requirements in the [issue](https://github.com/grafana/grafana/issues/58626): `Remove the heading/title Raw query.`

**Comment by bohandley** [comment]
As part of [G10 Getting started with Prometheus](https://github.com/grafana/grafana/issues/57754)

Thanks @gtk-grafana!

**Pull request merged**